proposals/lip-0045.md

```diff
@@ -234,6 +234,21 @@ In this section, we specify the substores that are part of the Interoperability
 | `MIN_CHAIN_NAME_LENGTH` | uint32 | 1 | The minimum length of a string specifying the name of a chain. |
 | `MAX_CHAIN_NAME_LENGTH` | uint32 | 32 | The maximum length of a string specifying the name of a chain. |
 
+#### Empty Cross-chain Message
+
+```java
+EMPTY_CCM = {
+    "module": "",
+    "crossChainCommand": "",
+    "nonce": 0,
+    "fee": 0,
+    "sendingChainID": EMPTY_BYTES,
```

Comment:
good point. @shuse2 Do you have any input here? How should we exactly specify the empty schema used for the invalid validation event?

Reply:
I think this is only used in event, and for the event, we don't call validation since it's not receiving from external.
As long as it's only used in the event, it should be fine or we could define in the event as well?